src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -25,12 +41,846 @@
      */
     @Test
     public void is_ignoreMissingFiles_shown_right() {
-        FreeStyleJob seed = jenkins.jobs.create(FreeStyleJob.class);
-        JobDslBuildStep jobdsl = seed.addBuildStep(JobDslBuildStep.class);
-        assertTrue(jobdsl.isIgnoreMissingFilesShown());
-        jobdsl.clickUseScriptText();
-        assertFalse(jobdsl.isIgnoreMissingFilesShown());
-        jobdsl.clickLookOnFilesystem();
-        assertTrue(jobdsl.isIgnoreMissingFilesShown());
+        FreeStyleJob seedJob = createSeedJob();
+        JobDslBuildStep jobDsl = seedJob.addBuildStep(JobDslBuildStep.class);
+        assertThat(jobDsl.isIgnoreMissingFilesShown(), is(true));
+        jobDsl.clickUseScriptText();
+        assertThat(jobDsl.isIgnoreMissingFilesShown(), is(false));
+        jobDsl.clickLookOnFilesystem();
+        assertThat(jobDsl.isIgnoreMissingFilesShown(), is(true));
+    }
+
+    /**
+     * Verifies that all configurations, done on the job configuration page,
+     * are saved correctly.
+     */
+    @Test
+    public void should_save_configurations() {
```

Comment:
Yes, there are additional plugins needed.
We fixed it in the pull request #333  of my team member.